docs/tutorials/morpheusvm/morpheusvm.md

```diff
@@ -257,10 +266,12 @@ func FeeKey() (k []byte) {
 Next, we'll define the `BalanceKey` function. `BalanceKey` will return the
 state key that stores the provided address' balance.
 
-The HyperSDK requires that keys include a "chunk suffix", which specifies
-the maximum size of the value that will ever be stored at that key. This also
-means that trying to change the suffix will change the key itself, so a write
-to the same key with a different suffix will write to a different location
+The HyperSDK requires using [size-encoded storage keys](../../explanation/features.md#size-encoded-storage-keys),
```

Comment:
Link to full explanation and make it more explicit the difference between the byte size of the chunk suffix itself (2 bytes for uint16) and the cap on the corresponding value in the key-value pair